### PR TITLE
Update spring-cloud-config.adoc

### PR DESCRIPTION
**Pull request by dmikusa-pivotal**
Add some detail around using `--data-urlencode` correctly & using the `-s` option to silence curl stats.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@dmikusa-pivotal Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=1636)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=1636) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@dmikusa-pivotal Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=1636)!

File: docs/src/main/asciidoc/spring-cloud-config.adoc [hunk 1]
```diff
@@ -1163,25 +1163,22 @@ The server also exposes `/encrypt` and `/decrypt` endpoints (on the assumption t
 If you edit a remote config file, you can use the Config Server to encrypt values by POSTing to the `/encrypt` endpoint, as shown in the following example:
 
 ----
-$ curl localhost:8888/encrypt -d mysecret
+$ curl localhost:8888/encrypt -s -d mysecret
 682bc583f4641835fa2db009355293665d2647dade3375c0ee201de2a49f7bda
 ----
 
-NOTE: If the value you encrypt has characters in it that need to be URL encoded, you should use the `--data-urlencode` option to `curl` to make sure they are encoded properly.
+TIP: If you testing with curl, then use `--data-urlencode` (instead of `-d`) and prefix the value to encrypt with `=` (curl requires this) or set an explicit `Content-Type: text/plain` to make sure curl encodes the data correctly when there are special characters ('+' is particularly tricky).
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-config.adoc:1170]: If you are testing...

File: docs/src/main/asciidoc/spring-cloud-config.adoc [hunk 2]
```diff
@@ -1163,25 +1163,22 @@ The server also exposes `/encrypt` and `/decrypt` endpoints (on the assumption t
...
 
-TIP: Be sure not to include any of the curl command statistics in the encrypted value.
-Outputting the value to a file can help avoid this problem.
+TIP: Be sure not to include any of the curl command statistics in the encrypted value, this is why the examples use the `-s` option to slience them. Outputting the value to a file can help avoid this problem.
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-config.adoc:1172]: silence

**Pull request closed without merging**